lib/imgix/client.rb

```diff
@@ -14,6 +14,10 @@ def initialize(options = {})
 
       @host = options[:host]
       validate_host!
+      warn "Warning: The identifier `host' has been deprecated and " \
```

Comment:
Just want to refer to my suggestion from earlier:

> Looking at the process we followed in [imgix-core-js](https://github.com/imgix/imgix-core-js/pull/44), we went ahead and added the `domain` field as part of the minor bump. This (in addition to the warning) gave users a chance to more-smoothly transition from using `host` to using `domain` before the major release. If they wanted to get rid of the warnings, they could refactor their code to using the new field name. I think this a lot better than throwing a warning and not enabling devs to do anything about it.

I think it would be a good idea to add the `domain` field now, add some logic to only throw the warning if `host` is used, and reassign `host` to `domain` (if used) for the sake of saving a bunch of time on refactoring. 
At least that way, users can resolve the warning ahead of the major bump.

Reply:
To clarify, you want all references to `@host` and `:host` to be replaced with `@domain`and `:domain` in this PR and handle `@host` and `:host` by `warn`ing `host` has been deprecated, then reassign host to domain?

Then in a subsequent PR, remove the warning and the references to `host` entirely?

Reply:
1. Not replaced, we want to expose a new field `domain` while leaving `host` there
2. If `host` is true, throw a `warn`
3. If `domain` is true, set `host = domain` at init and let the rest of the logic flow as usual

> Then in a subsequent PR, remove the warning and the references to host entirely?

This will be after the minor bump, before the major. Hopefully that clears things up!

Reply:
You got it!